tst/regression/CMakeLists.txt

```diff
@@ -53,6 +53,13 @@ list(APPEND TEST_ARGS "--driver ${CMAKE_BINARY_DIR}/example/advection/advection-
 --driver_input ${CMAKE_CURRENT_SOURCE_DIR}/test_suites/advection_convergence/parthinput.advection \
 --num_steps 25")
 
+# Performance regression test
+list(APPEND TEST_DIRS advection_performance)
+list(APPEND TEST_PROCS "1")
+list(APPEND TEST_ARGS "--driver ${CMAKE_BINARY_DIR}/example/advection/advection-example \
+--driver_input ${CMAKE_CURRENT_SOURCE_DIR}/test_suites/advection_performance/parthinput.advection_performance \
+--num_steps 5")
```

Comment:
This does not seem consistent with the documentation. It looks like we have to manually add tests to `cmake`?

Reply:
Yes. Good point. I'll update the doc.